types/office-js-preview/index.d.ts

```diff
@@ -19932,6 +21345,237 @@ declare namespace Excel {
     function run<T>(context: OfficeExtension.ClientRequestContext, batch: (context: Excel.RequestContext) => Promise<T>): Promise<T>;
     function postprocessBindingDescriptor(response: any): any;
     function getDataCommonPostprocess(response: any, callArgs: any): any;
+    /**
+     * Represents an `AllowEditRange` object found in a worksheet.
```

Comment:
Could we make this a little more descriptive? I'm not sure when I'd want this object or what it really is.